apps/ledger-live-mobile/src/screens/Onboarding/steps/welcome.tsx

```diff
@@ -207,7 +207,9 @@ function OnboardingStepWelcome({ navigation }: NavigationProps) {
           >
             {t("onboarding.stepWelcome.start")}
           </Button>
-          {recoverFeature?.enabled && recoverFeature?.params?.onboardingLogin ? (
```

Comment:
`onboardingLogin` doesn't exist in the current params. I don't know what it was referring to, so I took a guess, correct me if I'm wrong.